shell/packages/sandstorm-ui-powerbox/powerbox-client.js

```diff
@@ -161,6 +161,28 @@ SandstormPowerboxRequest = class SandstormPowerboxRequest {
           cardData.title = "Admin: grant all outgoing network access";
         } else if (cardData.frontendRef.ipInterface) {
           cardData.title = "Admin: grant all incoming network access";
+        } else if (cardData.frontendRef.emailVerifier) {
+          const services = cardData.frontendRef.emailVerifier.services;
+          if (services) {
+            const name = services[0];
```

Comment:
Weird that this is a list, but we only ever look at the first element of said list.  Does this make more sense as an optional String instead?